### ISSUE TITLE
Can't find jEdit Linter Panel

### ISSUE THREAD

**Issue by firefighterduck** [isabelle-prover/isabelle-linter#25]
tl;dr: I can't find the linter panel mentioned in the README.

I followed the setup section of the README and added the Isabelle2021-1-v1.0.0 release to my Isabelle2021-1 installation. The Isabelle plugin options contain an entry about the linter, but I can't find the linter panel (or see any other form of feedback from the linter whatsoever, even for specifically discouraged examples with the `all` bundle). The linter panel doesn't appear in the panel list under `Plugins > Isabelle` and it also doesn't seem to available elsewhere. In contrast to this, the CLI for the linter works as expected (yet is very slow and requires >10GB of RAM).

Is the linter panel available somewhere that I wouldn't expect it or does it maybe require some other kind of setup steps?

**Comment by firefighterduck** [comment]
> @firefighterduck how are you launching jedit? If you use the packaged executable, it wouldn't work. You need to use the `isabelle jedit` command (you can find the executable in `bin`)

Ah yes, that was the problem. Thanks for the quick answers. I wonder what the difference is between these two ways of launching jedit.
